src/main/java/rs/hybridit/serviceImpl/BookRentServiceImpl.java

```diff
@@ -78,6 +81,36 @@ public BookCopy getBookCopy(Book book) {
 		});
 	}
 
+	@Override
+	public List<ReportFrequency> getRentingStatistics() {
+		List<Book> books = bookRepository.findAll();
```

Comment:
But if I make method List<Book> findAllOrderByRentingCounterDesc in BookRepository I get 
Caused by: java.lang.IllegalArgumentException: Failed to create query for method public abstract java.util.List rs.hybridit.repository.BookRepository.findAllOrderByRentingCounter()! No parameter available for part rentingCounter SIMPLE_PROPERTY (1): [Is, Equals] NEVER.

I googled it and it says that this if you use OrderBy it can't be applied on findAll, but only to findBy(Something).

There are also some data jpa methods but they seem too complicated :'D

Reply:
There is something like this:
 @Override
    public List<Student> findAll() {
        return studentDao.findAll(sortByIdAsc());
    }

    private Sort sortByIdAsc() {
        return new Sort(Sort.Direction.ASC, "id");
    }